[2024-04-08T05:06:54.580+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-25T00:00:00+00:00 [queued]>
[2024-04-08T05:06:54.594+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-25T00:00:00+00:00 [queued]>
[2024-04-08T05:06:54.595+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-04-08T05:06:54.595+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2024-04-08T05:06:54.596+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-04-08T05:06:54.621+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): second_task> on 2021-11-25 00:00:00+00:00
[2024-04-08T05:06:54.627+0000] {standard_task_runner.py:55} INFO - Started process 9316 to run task
[2024-04-08T05:06:54.630+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2021-11-25T00:00:00+00:00', '--job-id', '1231', '--raw', '--subdir', 'DAGS_FOLDER/our_firest_dag.py', '--cfg-path', '/tmp/tmp1u32laj3']
[2024-04-08T05:06:54.632+0000] {standard_task_runner.py:83} INFO - Job 1231: Subtask second_task
[2024-04-08T05:06:54.707+0000] {task_command.py:388} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-25T00:00:00+00:00 [running]> on host 3dc1147ef06e
[2024-04-08T05:06:54.803+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Jiseungmin
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-25T00:00:00+00:00
[2024-04-08T05:06:54.804+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-04-08T05:06:54.806+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello i am task2 and this is the task2 task!']
[2024-04-08T05:06:54.814+0000] {subprocess.py:86} INFO - Output:
[2024-04-08T05:06:54.816+0000] {subprocess.py:93} INFO - Hello i am task2 and this is the task2 task!
[2024-04-08T05:06:54.816+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-08T05:06:54.845+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20211125T000000, start_date=20240408T050654, end_date=20240408T050654
[2024-04-08T05:06:54.882+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-04-08T05:06:54.904+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
